### DISCUSSION TITLE
parameters frontend error

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1882] [Q&A]
![Screenshot (47)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/948e4af6-72ea-442c-a8c9-da29ac77abda)

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/2bff6827-6710-4aa8-9acc-ad84c69b05ae)
